src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/ClientConfigPaths.cs

```diff
@@ -35,6 +35,7 @@ private ClientConfigPaths(string exePath, bool includeUserConfig)
 
```

Comment:
> `100644 → 100755`

This change is setting executable bit on all files you have touched. It should be undone.